proof/refine/ARM/Finalise_R.thy

```diff
@@ -4289,6 +4358,40 @@ lemma sched_context_maybe_unbind_ntfn_corres:
      apply (wpsimp wp: get_simple_ko_wp getNotification_wp split: option.splits)+
   done
 
+lemma replyRemove_corres:
+  "corres dc (invs and tcb_at tptr) (invs' and K (rptr' = rptr))
+      (reply_remove tptr rptr) (replyRemove rptr' tptr)"
+  apply (clarsimp simp: reply_remove_def replyRemove_def)
+  sorry
```

Comment:
This should work fine with what I have as work-in-progress. I really should finish removing this sorry...